cosmic/sample/sampler/independent.py

```diff
@@ -455,7 +456,7 @@ def sample_ecc(self, ecc_model='thermal', size=None):
             ecc = rndm(a=0.001, b=0.9, g=-0.42, size=size) 
```

Comment:
Still wondering about the choice of eccentricity here. Why are we using -0.42  as  default rather  than -0.45?

Reply:
I'm following eq 3 of De Mink & Belczynski 2015: https://iopscience.iop.org/article/10.1088/0004-637X/814/1/58/pdf

Reply:
If we want to adopt their values, then I think we should do the same for the orbital period (Sana's  best fit is 0.55, they use 0.5 for their standard assumption). I think it's clear to just use the best-fit values from Sana, even though there is a spread  for what could  be considered viable. What do you think?

Reply:
I think the best fit for the eccentricity is -0.42. I'm looking at page 8 of the supplement in the second paragraph. eta is the power law that they fit for eccentricity and pi is the power law for orbital period.

Reply:
From the Sana paper,  pg 8: 

"We generate 40 synthetic populations that share the properties of our sample
and are randomly drawn from the best-fit parent distributions, i.e. fbin = 0.69, π = -0.55, κ
= -0.10, and η = -0.45. Each synthetic population is input to our code and we re-derive
the best-fit intrinsic binary fraction and parent distributions. The mean and 1σ dispersion
of the retrieved parameters serve as a check for the presence of systematic biases and as a
probe for the accuracy of the procedure. We obtain <fbin> = 0.74 ± 0.09, <π> = -0.38 ±
0.22, <κ> = -0.21 ± 0.58 and <η> = -0.42 ± 0.17. "

The best-fit parameters are the first numbers,  the recovered parameters from their 40 realizations are the second set of numbers. They're trying (poorly) to prove that there aren't systematic  biases. But the best-fit parameters  are still the top set of numbers.

Reply:
ah! So I was misunderstanding. I think the De Mink & Belczynski paper has this wrong them as well. I really should have taken a stats class 🙃

I'll fix this right away!

Reply:
Thanks for sticking this out @michaelzevin -